src/ngraph_version_utils.h

```diff
@@ -23,5 +23,5 @@
    ((TF_MAJOR_VERSION == REQ_TF_MAJ_VER) &&            \
     (TF_MINOR_VERSION >= REQ_TF_MIN_VER)))
 
-#define TF_VERSION_GEQ_1_11 TF_VERSION_GEQ(1, 11)
+#define TF_ENABLE_DEADNESS_CHECK TF_VERSION_GEQ(1, 11)
```

Comment:
If we go by my earlier recommendation of using NGRAPH_TF_DISABLE_DEADNESS_CHECK then this line won't be needed.